src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -651,6 +652,22 @@ private string[] GetResourceKeys(CancellationToken ct)
 			return resourceKeys;
 		}
 
+		private string RewriteResourceKeyName(StringBuilder builder, string keyName)
```

Comment:
Consider making `static` if it doesn't reference instance members. [CA1822](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1822)